test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,23 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNotNull() {
```

Comment:
You can check for `elements contain null` as well. This method despised its name is not restricted to checking if `elements are not null` or not (take a look at the `elementsAreUnique` method above for reference) :)